src/wasm-traversal.h

```diff
@@ -537,6 +537,49 @@ struct ExpressionStackWalker : public PostWalker<SubType, VisitorType> {
   }
 };
 
+// Traversal keeping track of try depth
+
+template<typename SubType, typename VisitorType = Visitor<SubType>>
```

Comment:
Can you add a comment saying that if we ever add any other kind of handler scope beyond try-catch, we should handle that here as well?